src/include/sof/wait.h

```diff
@@ -166,7 +166,7 @@ static inline void wait_delay(uint64_t number_of_clks)
 
 static inline int poll_for_completion_delay(completion_t *comp, uint64_t us)
 {
-	uint64_t tick = clock_us_to_ticks(CLK_CPU, us);
+	uint64_t tick = clock_us_to_ticks(CLK_SSP, us);
```

Comment:
Instead of explicitly setting clock to SSP, maybe you should define new PLATFORM_DEFAULT_CLOCK.
PLATFORM_SCHED_CLOCK and PLATFORM_WORKQ_CLOCK can also be set to this new PLATFORM_DEFAULT_CLOCK.

Reply:
@RanderWang and we should also have some doxygen comments for each new PLATFORM_ macro so users know what it does...